content/pages/why.md

```diff
@@ -0,0 +1,89 @@
+Title: Why SaltyRTC?
+Date: 2016-12-03 19:00
+Modified: 2016-12-03 19:00
+Sort-Order: 1
+Summary: What is SaltyRTC all about and why should I use it?
+
+# Short
+
+Here's why you should use SaltyRTC. With SaltyRTC,
+
+* signalling data is fully end-to-end encrypted,
+* zero trust towards the server is needed,
+* no data is stored on the server,
+* servers can be self-hosted,
+* it doesn't fall apart without SSL/TLS, and
```

Comment:
Yes, but it does and should use TLS by default. By the way, I think you can drop "SSL". It's legacy.